src/app/SharpRaven/Data/JsonPacket.cs

```diff
@@ -396,5 +397,10 @@ private void Initialize(Exception exception)
                 }
             }
         }
+
+        public override int GetHashCode()
+        {
+            return this.ToString().GetHashCode()*17;
```

Comment:
I guess is because is a prime number but I don't know if is really necessary. @rolandh did you get any issue when you didn't use it?